README.md

```diff
@@ -24,28 +22,27 @@ Use this workflow to perform basic Go linting checks:
 on:
   push:
     branches:
-      - 'main'
```

Comment:
Why these changes? Also single-quoted YAML is safer.

Reply:
Errrr, my formatter must have done this for markdown.....I'll fix.